invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -194,10 +194,13 @@ export const RDMRecordFacetsValues = ({
           </Label>
         </List.Content>
         {hasChildren ? (
-          <i
-            className={`angle ${isActive ? "down" : "right"} icon`}
+          <button
+            className={`iconhold`}
```

Comment:
minor: I think this should be enough
```suggestion
            className="iconhold"
```